Test failures with the latest importlib-metadata 8.7.0

When I run the testsuite, two tests fail and it seems the reason is the incompatibility with the latest importlib-metadata:

```
=================================== FAILURES ===================================
___________________________ test_adding_entry_point ____________________________

    def test_adding_entry_point():
        original_entry_points = entry_points(group="docstring_to_markdown")
>       mock_entry_point = MockEntryPoint(
            name='high-priority-converter',
            group='docstring_to_markdown',
            value=HighPriorityConverter,
        )

tests/test_convert.py:111: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3.13/site-packages/importlib_metadata/__init__.py:214: in __init__
    self.module
/usr/lib/python3.13/site-packages/importlib_metadata/__init__.py:227: in module
    return self._match.module
/usr/lib64/python3.13/functools.py:1026: in __get__
    val = self.func(instance)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = EntryPoint(name='high-priority-converter', value=<class 'test_convert.HighPriorityConverter'>, group='docstring_to_markdown')

    @functools.cached_property
    def _match(self) -> _EntryPointMatch:
>       match = self.pattern.match(self.value)
E       TypeError: expected string or bytes-like object, got '_ProtocolMeta'

/usr/lib/python3.13/site-packages/importlib_metadata/__init__.py:239: TypeError
__________________________ test_replacing_entry_point __________________________

    def test_replacing_entry_point():
        assert convert(CPYTHON) == CPYTHON_MD
        original_entry_points = entry_points(group="docstring_to_markdown")
>       mock_entry_point = DistMockEntryPoint(
            name='cpython',
            group='docstring_to_markdown',
            value=CustomCPythonConverter
        )

tests/test_convert.py:123: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/lib/python3.13/site-packages/importlib_metadata/__init__.py:214: in __init__
    self.module
/usr/lib/python3.13/site-packages/importlib_metadata/__init__.py:227: in module
    return self._match.module
/usr/lib64/python3.13/functools.py:1026: in __get__
    val = self.func(instance)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = EntryPoint(name='cpython', value=<class 'test_convert.CustomCPythonConverter'>, group='docstring_to_markdown')

    @functools.cached_property
    def _match(self) -> _EntryPointMatch:
>       match = self.pattern.match(self.value)
E       TypeError: expected string or bytes-like object, got '_ProtocolMeta'

/usr/lib/python3.13/site-packages/importlib_metadata/__init__.py:239: TypeError
=========================== short test summary info ============================
FAILED tests/test_convert.py::test_adding_entry_point - TypeError: expected string or bytes-like object, got '_ProtocolMeta'
FAILED tests/test_convert.py::test_replacing_entry_point - TypeError: expected string or bytes-like object, got '_ProtocolMeta'
======================== 2 failed, 101 passed in 0.20s =========================
```

--- a/tests/test_convert.py
+++ b/tests/test_convert.py
@@ -75,7 +75,7 @@ class HighPriorityConverter(Converter):
 
 class MockEntryPoint(EntryPoint):
     def load(self):
-        return self.value
+        return getaattr(globals(), self.attr)
 
     dist = None
 
@@ -111,7 +111,7 @@ def test_adding_entry_point():
     mock_entry_point = MockEntryPoint(
         name='high-priority-converter',
         group='docstring_to_markdown',
-        value=HighPriorityConverter,
+        value="here:HighPriorityConverter",
     )
     with custom_entry_points([*original_entry_points, mock_entry_point]):
         assert convert('test') == 'HighPriority'
@@ -123,7 +123,7 @@ def test_replacing_entry_point():
     mock_entry_point = DistMockEntryPoint(
         name='cpython',
         group='docstring_to_markdown',
-        value=CustomCPythonConverter
+        value="here:CustomCPythonConverter",
     )
     with custom_entry_points([*original_entry_points, mock_entry_point]):
         assert convert('test') == 'test'

--- a/tests/test_convert.py
+++ b/tests/test_convert.py
@@ -75,7 +75,7 @@ class HighPriorityConverter(Converter):
 
 class MockEntryPoint(EntryPoint):
     def load(self):
-        return getaattr(globals(), self.attr)
+        return getattr(globals(), self.attr)
 
     dist = None
 

--- a/tests/test_convert.py
+++ b/tests/test_convert.py
@@ -75,7 +75,7 @@ class HighPriorityConverter(Converter):
 
 class MockEntryPoint(EntryPoint):
     def load(self):
-        return getattr(globals(), self.attr)
+        return globals()[self.attr]
 
     dist = None
 

